app/presenters/contacts_finder_presenter.rb

```diff
@@ -0,0 +1,101 @@
+class ContactsFinderPresenter
+
+  def initialize(contact_group)
+    @contact_group = contact_group
+    @organisation = contact_group.organisation
```

Comment:
This doesn't seem to use the contact_group anywhere, only the organisation.  It might be clearer to pass in the organisation here.